Chapter 11/package.json

```diff
@@ -19,10 +19,10 @@
   "license": "MIT",
   "dependencies": {
     "fastify": "^4.2.0",
-    "pino": "^7.6.5"
```

Comment:
I think we could remove it from the dep as we are using the fastify's pino version (v7)

I think it was added here because I was using fastify v3 with pino v7 at that time